apis/resourcecontrollerv2/v1alpha1/register.go

```diff
@@ -17,9 +17,8 @@ limitations under the License.
 package v1alpha1
 
 import (
-	"reflect"
-
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"reflect"
 	"sigs.k8s.io/controller-runtime/pkg/scheme"
```

Comment:
Our generator uses the "go fmt" ordering of imports, which is simple alpha order by package name.  We could also change this if necessary.

Reply:
the crossplane CI uses golangci-lint, which was complaining if the imports for the local project were not last and separated by a blank line. There are also some Crossplane conventions I noticed in the other projects, such as:
- Golang imports first
- <blank line separator>
- GitHub imports
- <blank line separator>
- Crossplane imports
- <blank line separator>
- Local project imports